src/UI/Art/SpriteArtist/MapTileDetailsArtist.java

```diff
@@ -101,7 +101,7 @@ private static void generateOverlay(GameMap map, XYCoord coord)
       if( unit.ammo >= 0 ) 
         unitAttrs.add(new AttributeArtist(SpriteLibrary.getAmmoIcon(), unit.ammo));
       if( unit.getCaptureProgress() > 0)
-        unitAttrs.add(new AttributeArtist(SpriteLibrary.getCaptureIcon(unit.CO.myColor), unit.getCaptureProgress()));
+        unitAttrs.add(new AttributeArtist(SpriteLibrary.getCaptureIcon(unit.CO.myColor), 20-unit.getCaptureProgress()));
```

Comment:
This smells magical to me

Reply:
It's magical in `Unit` as well. Where do you propose it should live instead?

Reply:
My "ideal" answer would be `GameRules`, but I feel like that'd be hard to set up access to.
Could be a static field in `TerrainType`?